package.json

```diff
@@ -14,7 +14,11 @@
     "cs:start": "npm-run-all --parallel cs:build:watch cs:browse",
     "cs:start:secure": "npm-run-all --parallel cs:build:watch cs:browse",
     "cs:dev:start": "npm-run-all --parallel cs:dev:build:watch cs:browse",
-    "cs:dev:start:secure": "npm-run-all --parallel cs:dev:build:watch cs:browse:secure"
+    "cs:dev:start:secure": "npm-run-all --parallel cs:dev:build:watch cs:browse:secure",
+    "nc:build": "cd neo-codap && npm run build",
```

Comment:
Do you want to add the `nc:build` task to the top level build task so both are built on a Travis run?

Reply:
I didn't originally because it's not strictly required for a top-level build, but it makes sense for validation purposes that the top-level build includes all builds, and one can always get a CollabSpace build with cs:build, so I think it makes sense. Assuming you agree, I can make that change, rebase, and merge the PR.

Reply:
For the build to complete successfully, I had to configure travis.yml to build the neo-codap node_modules folder in addition to the top-level node_modules, but once that was done everything worked as expected.